diff --git a/lib/sensu-plugin/utils.rb b/lib/sensu-plugin/utils.rb
index 8e9870b..ca638cd 100644
--- a/lib/sensu-plugin/utils.rb
+++ b/lib/sensu-plugin/utils.rb
@@ -42,6 +42,12 @@ module Sensu
           Net::HTTP::Put
         end
       end
+
+      # probably, something like "platform" in ohai will be needed.
+      # cf. lib/ohai/plugins/linux/platform.rb
+      def os
+        RUBY_PLATFORM.split("-")[1].sub(/\d+$/, '').downcase
+      end
     end
   end
 end
